README.md

```diff
@@ -15,395 +15,18 @@ Project design document: [Design for a CoinSwap Implementation for Massively Imp
 - [Teleport Transactions](#teleport-transactions)
   - [Contents](#contents)
   - [State of the project](#state-of-the-project)
-  - [How to create a CoinSwap on regtest with yourself](#how-to-create-a-coinswap-on-regtest-with-yourself)
-  - [How to create a CoinSwap on networks other than regtest](#how-to-create-a-coinswap-on-networks-other-than-regtest)
-  - [How to recover from a failed coinswap](#how-to-recover-from-a-failed-coinswap)
-  - [Developer resources](#developer-resources)
-    - [How CoinSwap works](#how-coinswap-works)
-    - [Notes on architecture](#notes-on-architecture)
-    - [Protocol between takers and makers](#protocol-between-takers-and-makers)
-    - [Code Structure](#code-structure)
-    - [Further reading](#further-reading)
+  - [Docs](#docs)
   - [Chris Belcher's personal roadmap for the project](#chris-belchers-personal-roadmap-for-the-project)
   - [Community](#community)
 
 ## State of the project
 
 The project is nearly usable, though it doesnt have all the necessary features yet. The code written so far is published for developers and power users to play around with. It doesn't have config files yet so you have to edit the source files to configure stuff. It is possible to run it on mainnet, but only the brave will attempt that, and only with small amounts.
```

Comment:
Maybe we can add some further information on what this library is currently. Along the line of, it's a cli app written in rust as demo prototype of working with underlying coinswap subroutines and primitives.